notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,318 @@
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "22a40144",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from rubicon_ml import Rubicon\n",
+    "\n",
+    "rubicon = Rubicon(persistence=\"memory\")\n",
+    "project = rubicon.get_or_create_project(\"Artifact Plots\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "4ccb7530",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>direction</th>\n",
+       "      <th>strength</th>\n",
+       "      <th>frequency</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>N</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>NNE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>NE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>ENE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>E</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  direction strength  frequency\n",
+       "0         N      0-1        0.5\n",
+       "1       NNE      0-1        0.6\n",
+       "2        NE      0-1        0.5\n",
+       "3       ENE      0-1        0.4\n",
+       "4         E      0-1        0.4"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.express as px\n",
+    "from plotly import data\n",
+    "\n",
+    "df = data.wind()\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "31e007bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_plot=px.scatter(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "#save our image for plotting\n",
```

Comment:
use markdown cells for text. these notebooks are rendered as examples in the documentation so text presents better in markdown cells than in code ones

Reply:
this is code to create the plot so I don't think we can do it in mark down

Reply:
Oh I see the comment? Yeah I'll move it to a more descriptive markdown cell